plugins/modules/grafana_datasource.py

```diff
@@ -469,9 +469,17 @@
 from ansible_collections.community.grafana.plugins.module_utils import base
 
 
-def compare_datasources(new, current, compareSecureData=True):
-    if 'uid' not in current:
-        del new['uid']
+def compare_datasources(new, current, compareSecureData=True, version=0):
+    if version >= 8:
```

Comment:
So I I understand this function, we want to delete `uid` from new and current if present.
Why not just test if uid is present in dict and remove it. No matter the version is.
```
if 'uid' in new:
  del new['uid']
if 'uid' in current:
  del current['uid']
```

Reply:
done